.editorconfig

```diff
@@ -0,0 +1,15 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+# Indentation style of 4 spaces
+[*]
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+
+# Windows-style newlines
+[*.bat]
+end_of_line = crlf
```

Comment:
Yes, I think they will execute correctly. But the default editor on most Windows systems for `.bat` files is Notepad, and Notepad requires lines to end with `crlf`. I suppose this is trivial and could go either way.